tests/test_parser.py

```diff
@@ -511,6 +511,19 @@ def test_term_extraction(self):
         simple_extracted = extract_query_terms(network_event + "| unique process_name, user_name\n\n| tail 10")
         self.assertListEqual(simple_extracted, [network_event.strip()])
 
+    def test_unicode_escape(self):
```

Comment:
can you add another test with rendered unicode such as `☢`

Reply:
Even though that should have always worked, apart from this update

Reply:
what do you mean by "rendered"? `String().render()`?

Reply:
```sql
process where process.name == "☢"
```

no biggie if not